### PR TITLE
plot_response degrees option

### PR DESCRIPTION
**Pull request by bonaime**
Option 'deg' for plotting plot_response in degrees (radian is default)
- [x] Tests

bode_degrees

### CODE REVIEW

**Comment by krischer** [comment]
Looks good but it does need a test case. Please generate the test image with the `v2.0.x` branch on matplotlib's github.

**Comment by megies** [comment]
Hmm.. OT.. maybe we should at some point consider using a nicer style as default as well, that image up there looks better than our default right now, I think (Axes, Tick setup etc)..

**Comment by krischer** [comment]
> Hmm.. OT.. maybe we should at some point consider using a nicer style as default as well, that image up there looks better than our default right now, I think (Axes, Tick setup etc)..

Our plots already look nicer on matplotlib 2 - we force the styles only for the tests.

**Comment by megies** [comment]
> Our plots already look nicer on matplotlib 2 - we force the styles only for the tests.

Ah! Didn't know.. nice!

**Comment by bonaime** [comment]
Image test for Response 
response_response_degrees

**Comment by QuLogic** [comment]
All the other parameters use full words; I think `deg` might be too short/unclear, maybe `plot_degrees` or something?

File: obspy/core/inventory/channel.py [hunk 1]
```diff
@@ -333,6 +333,8 @@ def plot(self, min_freq, output="VEL", start_stage=None, end_stage=None,
             amplitude/phase spectrum. If not specified, a new figure is opened.
         :type unwrap_phase: bool
         :param unwrap_phase: Set optional phase unwrapping using NumPy.
+        :type deg: bool
+        :param deg: if True plot bode in degrees
```
Comment (megies) [inline obspy/core/inventory/channel.py]: two back ticks for formatting:
```
``True``
```
in docstring will be displayed by sphinx as monotyped:

``True``

File: obspy/core/inventory/inventory.py [hunk 2]
```diff
@@ -728,6 +728,8 @@ def plot_response(self, min_freq, output="VEL", network="*", station="*",
             opened.
         :type unwrap_phase: bool
         :param unwrap_phase: Set optional phase unwrapping using NumPy.
+        :type deg: bool
+        :param deg: if True plot bode in degrees
```
Comment (megies) [inline obspy/core/inventory/inventory.py]: see above

File: obspy/core/inventory/response.py [hunk 3]
```diff
@@ -1294,6 +1294,8 @@ def plot(self, min_freq, output="VEL", start_stage=None,
             that should be plotted times two.
         :type unwrap_phase: bool
         :param unwrap_phase: Set optional phase unwrapping using NumPy.
+        :type deg: bool
+        :param deg: if True plot bode in degrees
```
Comment (megies) [inline obspy/core/inventory/response.py]: and again, backticks

File: obspy/core/inventory/station.py [hunk 4]
```diff
@@ -438,6 +438,8 @@ def plot(self, min_freq, output="VEL", location="*", channel="*",
             opened.
         :type unwrap_phase: bool
         :param unwrap_phase: Set optional phase unwrapping using NumPy.
+        :type deg: bool
+        :param deg: if True plot bode in degrees
```
Comment (megies) [inline obspy/core/inventory/station.py]: backticks..

**Comment by bonaime** [comment]
back ticks and plot_degrees are in the new commit

**Comment by krischer** [comment]
Travis failures are unrelated and dealt with in #1777.

**Comment by krischer** [comment]
Thanks for this!

**Pull request merged**